great_expectations/expectations/expectation.py

```diff
@@ -2278,7 +2278,7 @@ def get_validation_dependencies(
                 metric_value_kwargs=metric_kwargs["metric_value_kwargs"],
             )
 
-        if result_format_str in ["BASIC", "SUMMARY"]:
+        if result_format_str in ["BASIC"]:
```

Comment:
When getting `validation_dependencies` for `MapMetric` values, we were returning the dependencies too soon.

Reply:
I think we don't want to compute the unexpected_rows for the BASIC or SUMMARY cases: https://docs.greatexpectations.io/docs/reference/expectations/result_format/#fields-defined-for-column_map_expectation-type-expectations.

Reply:
Thank you @anthonyburdi, I may be incorrect here, but I think `unexpected_rows` can technically be calculated for both `BASIC` or `SUMMARY` (at least according to the docs)

When using `include_unexpected_rows`, you must explicitly specify result_format as well, and result_format must be more verbose than BOOLEAN_ONLY (which would be `SUMMARY`, `BASIC`, and `COMPLETE`)

Reply:
`https://docs.greatexpectations.io/docs/reference/expectations/result_format/` is where the statement is

Reply:
Thank you for clarifying, you are absolutely right.